### DISCUSSION TITLE
How to perform FindElements with Boa-constrictor

### DISCUSSION THREAD

**Discussion by annwojaczek** [q2ebanking/boa-constrictor#259] [Q&A] [answered]
I would like to return a list of column names.
I have a locator, that should return all columns:` #list-view-headers-row > th`
I located the element like this: `public static IWebLocator MethodsListViewPageColumns => new WebLocator("Method list view page columns", By.CssSelector("#list-view-headers-row > th"));`

Then I have a method, with which I want to find text of all column headers.
` public void ThenTheTableContainsAllColumns()
    {
        var columns = MethodsListViewPage.MethodsListViewPageColumns;
        var columnNames = Actor.AsksFor(Text.Of(columns));
    }`

The problem is, it is finding just first element. I guess it is just performing FindElement. 
How do I make it perform FindElements and return a list?

**Comment by annwojaczek** [comment, marked as answer]
In the end I changed Text.Of to TextList. It worked

**Comment by AutomationPanda** [comment]
Excellent! (Sorry for not answering sooner.)
